gradoop-examples/src/main/java/org/gradoop/examples/rollup/TelephoneCallGraph.java

```diff
@@ -0,0 +1,109 @@
+import org.gradoop.flink.model.impl.epgm.GraphCollection;
+import org.gradoop.flink.model.impl.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.operators.grouping.Grouping;
+import org.gradoop.flink.model.impl.operators.grouping.functions.aggregation.CountAggregator;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.gradoop.flink.util.GradoopFlinkConfig;
+
+/**
+ * A self contained example on how to use the rollUp operator on Gradoop's LogicalGraph class.
+ *
+ * The example uses our own telephone call example graph.
+ */
+public class TelephoneCallGraph {
+
+  /**
+   * Path to the example data graph
+   */
+  private static final String EXAMPLE_DATA_FILE =
+    TelephoneCallGraph.class.getResource("/data/gdl/tc.gdl").getFile();
+
+  /**
+   * Runs the program on the example data graph.
+   *
+   * The example provides an overview over the usage of the different rollUp methods.
+   * It showcases the application of rollUp functions that are already provided by
+   * Gradoop. Documentation for all available rollUp functions can be found in the projects wiki.
+   *
+   * @see <a href="https://github.com/dbs-leipzig/gradoop/wiki/Unary-Logical-Graph-Operators">
+   * Gradoop Wiki</a>
+   *
+   * Using the telephone call graph in the resources directory, the program will:
+   * 1. transform the long value of the "time" property into the separate properties "year",
+   * "month", "day", "hour" and "minute"
+   * 2. run a vertex based rollUp using the vertex label as well as the "country", "state", "city"
+   * properties
+   * 3. run an edge based rollUp using the "year", "month", "day", "hour" and "minute" properties
+   * 4. write both results into a separate GDL-File
+   *
+   * @param args arguments
+   */
+  public static void main(String[] args) throws Exception {
+    // init execution environment
+    ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+
+    // create loader
+    FlinkAsciiGraphLoader loader = new FlinkAsciiGraphLoader(GradoopFlinkConfig.createConfig(env));
+
+    // load data
+    loader.initDatabaseFromFile(
+      URLDecoder.decode(EXAMPLE_DATA_FILE, StandardCharsets.UTF_8.name()));
+
+    // get LogicalGraph representation of the telephone call graph
+    LogicalGraph callGraph = loader.getLogicalGraph();
+
+    // create year, month, day, hour and minute properties out of the time stamp
+    callGraph = callGraph.transformEdges(new TimePropertyTransformationFunction<>("time"));
+
+    // path prefix for dot file
+    String pathPrefix = System.getProperty("user.home") + "/telephoneGraph";
+
+    // group by rollUp on vertices
+    String gdlPath = pathPrefix + "_vertexRollUp.gdl";
+    GraphCollection graphGroupedByRollUpOnVertices =
+      callGraph.groupVerticesByRollUp(
+      Arrays.asList(Grouping.LABEL_SYMBOL, "country", "state", "city"),
+      Collections.singletonList(new CountAggregator("count")),
+      Collections.emptyList(),
```

Comment:
Without any edge grouping key, superEdges have no label etc. Use at least label and maybe year and month.